pkgs/games/super-tux-kart/default.nix

```diff
@@ -100,15 +104,13 @@ stdenv.mkDerivation rec {
     harfbuzz
     mcpp
     wiiuse
-    angelscript
   ];
 
   cmakeFlags = [
     "-DBUILD_RECORDER=OFF" # libopenglrecorder is not in nixpkgs
-    "-DUSE_SYSTEM_ANGELSCRIPT=OFF" # doesn't work with 2.31.2 or 2.32.0
+    "-DUSE_SYSTEM_ANGELSCRIPT=OFF" # doesn't work with our 2.35.0 on aarch64-linux
     "-DCHECK_ASSETS=OFF"
     "-DUSE_SYSTEM_WIIUSE=ON"
-    "-DUSE_SYSTEM_ANGELSCRIPT=ON"
```

Comment:
this is also removing for x86, should we just use a `optionals stdenv.isAarch64` to opt-out of the cmake flags?

Reply:
Hope it's better now.